cinema/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+from .models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255, required=True)
+    description = serializers.CharField(max_length=255, required=True)

```

Comment:
The `description` field in the serializer has a `max_length` of 255, which does not match the `TextField` type in the `Movie` model. `TextField` does not enforce a maximum length, so you might want to remove the `max_length` constraint from the serializer or ensure consistency with the model.